hasura/graphql-engine
# console: unique filenames for exported metadata (close #1772)

### Description



Generate metadata filenames such as: `hasura_metadata_2020_03_13_18_10_31_711.json`

### Changelog


### Affected components




### Related Issues



#1772

### Solution and Design



### Steps to test and verify


Head to `/settings/metadata-actions` page and hit `Export Metadata`. Observe downloaded file filename

### Limitations, known bugs & workarounds
